README.md

```diff
@@ -20,7 +20,12 @@ Use the following command to start DataLens containers:
 
 ```bash
 git clone https://github.com/datalens-tech/datalens && cd datalens
+
+# with demo database
 HC=1 docker compose up
+
+# with persistent database
```

Comment:
we will use persistent database by default (#22), so we should choose a different word here. Maybe "external" database?

Reply:
done